mlflow/utils/_capture_modules.py

```diff
@@ -84,6 +85,11 @@ def main():
 
     cap_cm = _CaptureImportedModules()
 
+    if flavor == "spark" and is_in_databricks_runtime():
+        from dbruntime.spark_connection import initialize_spark_connection
```

Comment:
This approach breaks when `initialize_spark_connection` is renamed or moved to a different module.

Reply:
Should we make the spark initialization process modifiable via patching or an environment variable?

Reply:
Add a MLR test to prevent it break.